app/src/main/java/fr/free/nrw/commons/upload/depicts/DepictsRoomDataBase.kt

```diff
@@ -0,0 +1,37 @@
+package fr.free.nrw.commons.upload.depicts
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import androidx.room.TypeConverters
+
+@Database(entities = arrayOf(Depicts::class), version = 1, exportSchema = false)
+@TypeConverters(depictsRoomDataBaseConverter::class)
+abstract class DepictsRoomDataBase : RoomDatabase() {
+
+    abstract fun DepictsDao(): DepictsDao
+
+    companion object {
+        // Singleton prevents multiple instances of database opening at the
+        // same time.
+        @Volatile
+        private var INSTANCE: DepictsRoomDataBase? = null
+
+        fun getDatabase(context: Context): DepictsRoomDataBase {
+            // if the INSTANCE is not null, then return it,
+            // if it is, then create the database
+            return INSTANCE ?: synchronized(this) {
+                val instance = Room.databaseBuilder(
+                    context.applicationContext,
```

Comment:
to get the context here, you have passed application at the beginning of method calls. No need to pass Application, just pass the context.

Reply:
Why do we need to create a new Database? Why are we not using the existing database and adding a table in that?